test/Microsoft.Net.Http.Headers.Tests/MediaTypeHeaderValueTest.cs

```diff
@@ -39,6 +40,76 @@ public void Ctor_MediaTypeInvalidFormat_ThrowFormatException()
             AssertFormatException("text/plain;charset=utf-8"); // ctor takes only media-type name, no parameters
         }
 
+        public static IEnumerable<string[]> MediaTypesWithSuffixes
+        {
+            get
```

Comment:
You have the unique opportunity to earn @pranavkm points by doing 

```C#
public static IEnumerable<string[]> MediaTypesWithSuffixes =>
    new List<string []>{ ... }
```

There is also a TheoryData type on Xunit that makes it a bit easier (you save the new[] and it's more strongly typed)